1-js/05-data-types/02-number/article.md

```diff
@@ -1,20 +1,20 @@
 # Numbers
 
-All numbers in JavaScript are stored in 64-bit format [IEEE-754](http://en.wikipedia.org/wiki/IEEE_754-1985) also known as "double precision".
+All numbers in JavaScript are stored in 64-bit format [IEEE-754](http://en.wikipedia.org/wiki/IEEE_754-1985), also known as "double precision".
 
-Let's recap what we know about them and add a little bit more.
+Let's recap and expand upon what we currently know about them.
 
 ## More ways to write a number
 
-Imagine, we need to write a billion. The obvious way is:
+Imagine we need to write 1 billion. The obvious way is:
 
 ```js
 let billion = 1000000000;
 ```
 
-But in real life we usually dislike writing many zeroes. It's easy to mistype. Also we are lazy. We will usually write something like `"1bn"` for a billion or `"7.3bn"` for 7 billions 300 millions. The similar is true for other big numbers.
+But in real life we usually avoid writing the full number as it's easy to mistype. Also, we are lazy. We will usually write something like `"1bn"` for a billion or `"7.3bn"` for 7 billion 300 million. The same is true for most large numbers.
```

Comment:
The idea is that it's easy to type wrong many **zeroes**, not just full numbers. What's wrong about explicitly saying it here?

Reply:
P.S. Maybe the last sentence here should be removed?

Reply:
@iliakan  "writing many zeroes" is a somewhat awkward phrase, but you are right that it is more explicit than "the full number". In @gwooly edit, the reader must infer that "the full number" refers to the billion variable value, but to me it makes sense. alternate suggestion "we usually avoid writing a long string of zeroes as it's easy to mistype".